File: 2014-10-CSharp-Fundamentals-1/7. Arrays/Reverse/Reverse.cs
using System;

class Reverse
{
	static void Main()
	{
		int[] array = new int[] {1, 2, 3, 4, 5};

        // Get array size
        int length = array.Length;

        // Declare and create the reversed array
        int[] reversed = new int[length];
        
		// Initialize the reversed array
		for (int index = 0; index < length; index++)
		{
			reversed[length-index-1] = array[index];
		}

		// Print the reversed array elements
		for (int index = 0; index < length; index++)
		{
			Console.WriteLine(reversed[index]);
		}
	}
}

File: 2015-10-Java-Fundamentals-1/07. Arrays/demos/Reverse.java
import java.text.MessageFormat;

public class Reverse {
    public static void main(String[] args) {
        int[] array = new int[] { 1, 2, 3, 4, 5 };

        // Get array size
        int length = array.length;

        // Declare and create the reversed array
        int[] reversed = new int[length];

        // Initialize the reversed array
        for (int index = 0; index < length; index++) {
            reversed[length - index - 1] = array[index];
        }

        // Print the reversed array elements
        for (int index = 0; index < length; index++) {
            writeLine(reversed[index]);
        }
    }

    public static void writeLine(Object obj) {
        System.out.println(obj);
    }
}